cumulus/steps/dev_tools/code_build_action.py

```diff
@@ -155,7 +155,7 @@ def create_project(self, chain_context, codebuild_role, codebuild_environment, n
             DependsOn=codebuild_role,
             Artifacts=artifacts,
             Environment=codebuild_environment,
-            Name=project_name,
+            Name="%s-%s" % (chain_context.instance_name, project_name),
```

Comment:
This one is different too.  It has a dash. 

I'm sure there are a bunch of other inconsistencies.. so not asking for a change in this PR, but I'll create an issue on this board to address it.